js/stdlib/ts/shared_impl.ts

```diff
@@ -345,14 +346,15 @@ export const stdContract =
   const _initialize = () => {
     const {
       getContractInfo, getContractAddress, getContractCompanion,
-      getBalance, sendrecv, recv, getCurrentStep, getState, apiMapRef
+      getBalance, sendrecv, recv, getCurrentStep, getState, apiMapRef,
+      simTokenAccepted
```

Comment:
If you wanted to be clever, you could make it so that this returns the `_` function and then you generate the one that does the `sim_txn` setting